### PR TITLE
Added the aboutLayer5 Section from the authors Page

### LINKED ISSUE #80: About Layer5 section needed in the author's section
#### Description

The Service mesh patterns site needs a about layer5 section on the about authors page according to figma design!!
#### Environment:
- Meshery Version: 
- Kubernetes Version: 
- Host OS:
- Browser:

### PR DESCRIPTION
**Pull request by eeshaanSA**
**Description**

This PR fixes #80 

**Notes for Reviewers**
Please suggest the changes that need to be made.
**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by eeshaanSA** [comment]
// @legendarykamal @debo19 @leecalcote

**Comment by legendarykamal** [comment]
@Nikhil-Ladha can you please start/run the workflows.

**Comment by l5io** [comment]
🚀 Preview for commit db3f43ea0a2b8dd2d4a18cac9aa3b64e7ca0389e at: https://633acc99bf4c255f00a340b5--service-mesh-patterns.netlify.app

**Comment by Nikhil-Ladha** [comment]
Hold this PR for sometime.

**Comment by eeshaanSA** [comment]
@Nikhil-Ladha @leecalcote @debo19 can you review this, and give feedback?

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]
Please rebase the code with the master and remove the duplicates from the PR i.e, the images and pages.
Also, you don't need to define the fonts again for a section it is already defined globally.

**Comment by eeshaanSA** [comment]
@Nikhil-Ladha pl enable the workflows.

**Comment by l5io** [comment]
🚀 Preview for commit e8d71d0a2c9f734c3850ee73dd0af6c9fd7489c3 at: https://63513bf6e53b071e0d85787e--service-mesh-patterns.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 3bc43c94cb5e9e74660df942a4a35a8ae7703cf1 at: https://635fb66e2c15d82307024a3f--service-mesh-patterns.netlify.app

**Comment by leecalcote** [comment]
@UtkarshMishra12 please review.

**Comment by UtkarshMishra12** [comment]
@Lee On clicking the [About](https://636d0222fce5d9231d9fc660--service-mesh-patterns.netlify.app/) button it's redirecting to the page: layer5.io/company/about, but I think as per your suggestion there should be a dedicated page for this.

https://user-images.githubusercontent.com/67385503/201134333-dad5f85e-ccf8-4188-9a9f-c0bd322ee516.mp4

**Comment by UtkarshMishra12** [comment]
@leecalcote  On clicking the [About](https://636d0222fce5d9231d9fc660--service-mesh-patterns.netlify.app/) button it's redirecting to the page: layer5.io/company/about, but I think as per your suggestion there should be a dedicated page for this.
(i.e In short I'm not able to see the new About page )

https://user-images.githubusercontent.com/67385503/201134333-dad5f85e-ccf8-4188-9a9f-c0bd322ee516.mp4

**Comment by eeshaanSA** [comment]
This makes sense according to me, @UtkarshMishra12. After this PR gets merged, there is going to be an about section just after the technical review, and not a different page.

**Comment by eeshaanSA** [comment]
So, if we are to put the button there, it has to be directed to the Layer5 site.

**Comment by UtkarshMishra12** [comment]
@eeshaanSA Oh sorry😅, I thought you have added a new page for the about page on the navbar.

**Review by UtkarshMishra12 — COMMENTED** [review]
Please make the "about section" responsive as per the screen size of the device.
Also the button "Join the Layer5 Community"  is not working. (It should redirect to Slack)

https://user-images.githubusercontent.com/67385503/201526791-4f468e5d-03b9-4787-bb0e-e2f04082e3e8.mp4

**Comment by l5io** [comment]
🚀 Preview for commit 11d0eb780cb5284feebb367d46a838f4ee92c4b5 at: https://6376ab1bb506891600af2a31--service-mesh-patterns.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 9644184b3884530cbc3761d1d5c9e891cc421e92 at: https://637734a11bb4ba0d9ceebb65--service-mesh-patterns.netlify.app

**Review by UtkarshMishra12 — COMMENTED** [review]
@eeshaanSA Now it's responsive as per the screen size, but I see when opening the site on mobile devices there is an irregular space in the para (see the video). Can you look at this?


https://user-images.githubusercontent.com/67385503/202657278-c19d5e28-fea7-4f35-8cf8-e840cd21f907.mp4

**Comment by eeshaanSA** [comment]
Any idea what could be causing this? @UtkarshMishra12 
The irregular spaces?

**Comment by eeshaanSA** [comment]
@Nikhil-Ladha can we have this merged?

File: site/src/sections/AboutLayer5/aboutl5io.style.js
Comment (Nikhil-Ladha) [inline site/src/sections/AboutLayer5/aboutl5io.style.js:4]:
```suggestion
  margin: 0.5rem auto 3rem;
  .outer-box {
```

File: site/src/sections/AboutLayer5/aboutl5io.style.js
Comment (Nikhil-Ladha) [inline site/src/sections/AboutLayer5/aboutl5io.style.js:12]:
```suggestion
```

File: site/src/sections/AboutLayer5/aboutl5io.style.js
Comment (Nikhil-Ladha) [inline site/src/sections/AboutLayer5/aboutl5io.style.js:109]:
```suggestion
      margin: 2rem auto;
```

File: site/src/sections/AboutLayer5/aboutl5io.style.js
Comment (Nikhil-Ladha) [inline site/src/sections/AboutLayer5/aboutl5io.style.js:71]:
```suggestion
```

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**
